### PR TITLE
Feature/error

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

- Add & implement Error component with graphic, css animation, & text
- Fix bug that hid PokemonGrid when getting error on PokemonDetais page

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/Error/Error.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  return (
+    <div className='error-container'>
+      <img
+        alt='Professor Oak'
+        className='prof-oak'
+        src='Images/prof-oak.png'
+      ></img>
+      <div className='error-message'>
+        <h2 className='error-quote'>"Everybody makes a wrong turn once in a while!"</h2>
+        <p className='error-note'>Try again. Enter a real Generation 1 Pokemon name or ID (1 to 152).</p>
```
Comment (Matt-Roden) [inline src/components/Error/Error.js]: Our app only has 1 -151, we'll need to change this text

File: src/components/Error/Error.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './Error.css';
+import { gsap, Power3 } from 'gsap';
```
Comment (Matt-Roden) [inline src/components/Error/Error.js]: We don't need this if we aren't using gsap for animations in this file

**Review by Matt-Roden — CHANGES_REQUESTED** [review]
I'm excited to see how this animation looks, professor oak is a good choice for an image!

If we aren't using gsap, we don't need to import it on the error file. Unless you plan on using it in the future, let's take it out

File: src/components/Error/Error.js (hunk 1 above)
Comment (joanafbrito) [reply src/components/Error/Error.js]: I think this is a very easy change! Would you mind changing that before we merge?

**Review by joanafbrito — CHANGES_REQUESTED** [review]
On the error file line 16 the ids are 1 - 151 ( once there's another request for changing would be great to change that before merging)
Ps: not sure why but it says that this branch has merging conflicts ?

File: src/components/Error/Error.js (hunk 1 above)
Comment (hayleyw7) [reply src/components/Error/Error.js]: Thanks, I'll do that. :)

File: src/components/Error/Error.js (hunk 2 above)
Comment (hayleyw7) [reply src/components/Error/Error.js]: Removed for now. Thank you!

File: src/components/Error/Error.js [hunk 3]
```diff
@@ -13,7 +12,7 @@ const Error = () => {
       ></img>
       <div className='error-message'>
         <h2 className='error-quote'>"Everybody makes a wrong turn once in a while!"</h2>
-        <p className='error-note'>Try again. Enter a real Generation 1 Pokemon name or ID (1 to 152).</p>
+        <p className='error-note'>Try again. Enter a real Generation 1 Pokemon name or ID (1 to 151).</p>
```
Comment (joanafbrito) [inline src/components/Error/Error.js:15]: Thank you changing that !

**Review by joanafbrito — APPROVED** [review]
Thank's for changing that and making sure there's no merging conflicts

**Review by Matt-Roden — APPROVED** [review]

**Pull request merged**